doc/Developing.md

```diff
@@ -3,12 +3,13 @@
 ## Setting up your development environment
 
 * Install JDK 8
-  * On mac with [homebrew]() installed:
+  * On mac with [homebrew](https://brew.sh/) installed:
     ```
     $ brew tap AdoptOpenJDK/openjdk
     $ brew cask install adoptopenjdk8
     # Use java_home to find the location of JAVA_HOME to set
     $ /usr/libexec/java_home -V
+    # Recommend adding the following line to ~/.zshrc
```

Comment:
Should it be "Recommend" or "Recommended"?

Reply:
Re-worded to "We recommend adding..."